app/styles/content.styl

```diff
@@ -0,0 +1,119 @@
+main
+    flex-grow: 1
+
+    @media (max-width: 1023px)
```

Comment:
Prefer use `em` units also for media queries, this way your UI remains responsive even when browser is zoomed. You can safely assume a base at 16px, so the query'll be: 

```styl
@media (max-width: 63.9375em)    // 1023/16 
```

Best resource about this best practice (in french): https://www.paris-web.fr/2013/conferences/un-petit-pas-pour-lem-un-grand-pas-pour-le-web.php